src/features/Apiexplorer/Schema/SchemaObjectContent.tsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import RecursiveProperties from './RecursiveProperties';
+import SchemaDescription from './SchemaDescription';
+import styles from './Schema.module.scss';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+// Header component
+type TSchemeBodyHeader = {
+  key_value: string;
+  type: string;
+  defaultValue: string;
+  pattern: string;
+  title: string;
+  is_open_object: boolean;
+  setIsOpenObject: (boolean) => void;
+  examples: string[];
+  enum;
+};
+type TSchemaObjectContent = {
+  key_value: string;
+  properties: any;
+};
+const SchemaBodyHeader = ({
+  key_value,
+  type,
+  defaultValue,
+  pattern,
+  examples,
+  enum: _enum,
+  title,
+  is_open_object,
+  setIsOpenObject,
+}: TSchemeBodyHeader) => {
+  let typeClassName;
+  switch (type) {
+    case 'number':
+      typeClassName = styles.number;
+      break;
+    case 'array':
+      typeClassName = styles.array;
+      break;
+    case 'integer':
+      typeClassName = styles.integer;
+      break;
+    default:
+      typeClassName = styles.string;
+      break;
+  }
+  if (Array.isArray(type)) {
+    type.map((item) => {
+      if (item !== 'null') {
+        switch (item) {
+          case 'number':
+            typeClassName = styles.number;
+            break;
+          case 'array':
+            typeClassName = styles.array;
+            break;
+          case 'integer':
+            typeClassName = styles.integer;
+            break;
+          default:
+            typeClassName = styles.string;
+            break;
+        }
```

Comment:
No because the other switch case only runs of recursion

Reply:
I suggest we can wrap the switch itself inside a function that returns the value. Then we call the function twice instead of having the exact same code duplicated beneath eachother.